Document operation modes for Twinkly integration (#36567)
home-assistant/home-assistant.github.io: source/_integrations/twinkly.markdown

## Effects

For devices with software version > 2.7.1, you can also control the effects on your Twinklys.

The Twinkly devices do not initially have any effects stored locally. Effects must be added from the Twinkly application before they become visible in Home Assistant.

Make sure the latest firmware is installed on your Twinkly devices and add effects by _apply_-ing them from the Twinkly app.

## Operation Modes

Modes can be switched using the select entity. 

For most use cases, "Color", "Movie" and "Playlist" modes are the most relevant.

**Color Mode**
- Displays a static color
- Controlled through the light entity
- Color can be set via RGB/RGBW

**Movie Mode**
- Plays effects previously uploaded through the Twinkly app
- Effects must be created and transferred to the device first using the app
- Effect selection is done through the light entity

**Playlist Mode**
- Sequential playback of multiple effects
- Playlist must be uploaded through the Twinkly app first

**Off Mode**
- Turns off the lighting

**Demo Mode**
- Shows predefined demo effects
- Primarily intended for presentation purposes
- Limited practical use

**Unsupported Modes:**

> - Effect Mode: predefined effects built into the device, you are probably better off using Movie Mode

Reviewer: ```suggestion
- Effect Mode: predefined effects built into the device. Alternatively, you can use Movie Mode
```
